### ISSUE TITLE
Core Graphics Text Color

### ISSUE THREAD

**Issue by dzhou121** [linebender/druid#1264]
It looks text color isn't correct with Core Graphics backend. The following simple example just draws some text with the same color with the background, but the actual text color is different with the background. 

```
    Label::new("test label")
        .with_text_color(Color::rgb8(64, 120, 242))
        .background(Color::rgb8(64, 120, 242))
```

<img width="801" alt="Screenshot 2020-09-28 at 09 43 06" src="https://user-images.githubusercontent.com/1169480/94410216-044bb480-016f-11eb-8775-113468e3e039.png">

**Comment by cmyr** [comment]
I... have no idea what's going on here.

**Comment by cmyr** [comment]
@dzhou121 it might be worth seeing if this is also the case when you use piet directly?

**Comment by dzhou121** [comment]
Do you mean this?

```
ctx.text()
    .new_text_layout("sample text")
    .text_color(Color::rgb8(64, 120, 242));
```

It has the same issue.

**Comment by cmyr** [comment]
I mean checking out the piet repo and modifying one of the sample images `piet/src/samples/*` and then running it from piet-coregraphics (`cargo run --example=test_picture N`)

**Comment by dzhou121** [comment]
It's the same if running it directly. 
![coregraphics-test-12](https://user-images.githubusercontent.com/1169480/94448739-2d863800-01a3-11eb-993d-529dea4f5807.png)

piet-cairo is fine
![cairo-test-12](https://user-images.githubusercontent.com/1169480/94448830-47c01600-01a3-11eb-964c-4fd9e2bfcfbc.png)

**Comment by cmyr** [comment]
cool, this might be something to do with text being drawn with a different blend mode? I'd have to experiment, but it looks like an issue in piet.

**Comment by dzhou121** [comment]
Can't see anything in piet that change blend mode though?

**Comment by cmyr** [comment]
no, it would be something specific in the coregraphics backend that we would need to fiddle with.
